### ISSUE TITLE
Multiple attributes reported as unknown functions

### ISSUE THREAD

**Issue by Mika-** [shipmonk-rnd/composer-dependency-analyser#123]
Multiple attributes are reported as unknown when using array notation.

```php
<?php
use Symfony\Component\Validator\Constraints as Assert;

class User {
    #[
        Assert\NotNull(message: 'Field is mandatory'),
        Assert\NotBlank(message: 'Field is mandatory'),
    ]
    private string $name;
}
```

This only seems to happen when attributes have parameters as `#[Assert\NotNull, Assert\NotBlank]` doesn't rise errors.

**Comment by janedbal** [comment]
Thank you! Should be fixed in `dev-master`, I'll release the fix soon.

**Comment by janedbal** [comment]
Fixed in `1.5.2`
